client/src/pages/Scheduler.js

```diff
@@ -51,26 +85,125 @@ const Scheduler = () => {
+                </Grid>
+              </Grid>
+              <Grid
+                direction="row"
+                alignItems="center"
+                container
+                justify="flex-start"
+                className={classes.inputRow}
+              >
+                <Grid xs="2" item>
+                  <InputLabel className={classes.label}>Email</InputLabel>
+                </Grid>
+
+                <Grid xs="10" item>
+                  <TextField
+                    variant="outlined"
+                    margin="normal"
+                    name="email"
+                    type="text"
+                    onChange={handleFormChange}
+                    fullWidth
+                    value={appointmentDetails.email}
+                  />
+                </Grid>
+              </Grid>
+              <Grid
+                direction="row"
+                alignItems="center"
+                container
+                justify="flex-start"
+                className={classes.inputRow}
+              >
+                <Grid xs="2" item>
+                  <InputLabel className={classes.descriptionLabel}>
+                    Notes
+                  </InputLabel>
+                </Grid>
+
+                <Grid xs="10" item>
+                  <TextField
+                    variant="outlined"
+                    margin="normal"
+                    name="notes"
+                    multiline
+                    onChange={handleFormChange}
+                    fullWidth
+                    value={appointmentDetails.notes}
+                    placeholder={`Write any details that will help prepare for our meeting.
+                                    
+                                    
+                                    `}
+                    className={classes.textArea}
+                  />
+                </Grid>
+              </Grid>
 
-          <Grid item container spacing={2}>
-            <Grid item xs={7}>
-              <PickDate
-                selectedDate={selectedDate}
-                setSelectedDate={setSelectedDate}
-              />
-              <Typography variant="subtitle2">
-                Coordinated Universal Time
-              </Typography>
-            </Grid>
-            <Grid item xs={5}>
-              <PickTime selectedDate={selectedDate} />
-            </Grid>
-          </Grid>
+              <Button
+                color="primary"
+                className={classes.button}
+                onClick={createAppointment}
+              >
+                Schedule Event
+              </Button>
+            </>
+          ) : (
+            <>
+              <Grid item>
+                <Typography variant="h5" className={classes.title}>
+                  Select a Date {"&"} Time
+                </Typography>
+              </Grid>
+              <Grid item container spacing={2}>
+                <Grid item xs={7}>
+                  <PickDate selectedDate={selectedDate} />
```

Comment:
```suggestion
                  <PickDate selectedDate={selectedDate} setSelectedDate={setSelectedDate} />
```
Changing the date throws an error because the calendar expects a setter function
<img width="1044" alt="Screen Shot 2021-01-26 at 2 14 15 PM" src="https://user-images.githubusercontent.com/48573314/105893185-c8bbdc80-5fe0-11eb-8065-6a1bed9ce7d0.png">